.github/workflows/test_devices_plugin.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Lint and Test Devices Plugin
+
+# on:
+#   push:
+#     branches: ["main"]
+#     paths:
+#     - plugins/devices/src/turnkeyml_plugin_devices/common/**
+#     - plugins/devices/src/turnkeyml_plugin_devices/onnxrt/**
+#     - plugins/devices/src/turnkeyml_plugin_devices/torchrt/**
+#     - plugins/devices/src/turnkeyml_plugin_devices/tensorrt/**
+#     - plugins/devices/setup.py
+#     - .github/workflows/test_devices_plugin.yml
+#   pull_request:
+#     branches: ["main"]
+#     paths:
+#     - plugins/devices/src/turnkeyml_plugin_devices/common/**
+#     - plugins/devices/src/turnkeyml_plugin_devices/onnxrt/**
+#     - plugins/devices/src/turnkeyml_plugin_devices/torchrt/**
+#     - plugins/devices/src/turnkeyml_plugin_devices/tensorrt/**
+#     - plugins/devices/setup.py
+#     - .github/workflows/test_devices_plugin.yml
```

Comment:
This will be enabled in a follow-up PR because it requires `turnkeyml==3.0.8` to exist.